beam-postgres/examples/src/Pagila/Schema/V0001.hs

```diff
@@ -177,7 +213,7 @@ data StaffT f
   , staffStore     :: PrimaryKey StoreT f
   , staffActive    :: Columnar f Bool
   , staffUsername  :: Columnar f Text
-  , staffPassword  :: Columnar f ByteString
+  , staffPassword  :: Columnar f Text  -- TODO use ByteString
```

Comment:
Why can't this be done?

Reply:
`staffPassword` is used here with the function `field`:
https://github.com/haskell-beam/beam/blob/719af70323f8f31cfcdb5398e4e961a48efc6d7b/beam-postgres/examples/src/Pagila/Schema/V0001.hs#L418

The `field` function is hard-coded to `Text`:
https://github.com/haskell-beam/beam/blob/719af70323f8f31cfcdb5398e4e961a48efc6d7b/beam-migrate/Database/Beam/Migrate/SQL/Tables.hs#L340-L343

Reply:
`staffPassword` is populated by a `binaryLargeObject`:
https://github.com/haskell-beam/beam/blob/719af70323f8f31cfcdb5398e4e961a48efc6d7b/beam-postgres/examples/src/Pagila/Schema/V0001.hs#L418

In `Database.Beam.Query.DataTypes`
https://github.com/haskell-beam/beam/blob/719af70323f8f31cfcdb5398e4e961a48efc6d7b/beam-core/Database/Beam/Query/DataTypes.hs#L102-L104

I could covert it with `encodeUtf8` or `encodeUtf16`, but the best solution would be to support `binaryLargeObject` `ByteString`, do you agree?

Reply:
Hmm I see, let's leave `staffPassword  :: Columnar f Text` for now.

I'm weary of breaking compatibility for users, so if someone has a concrete issue with `binaryLargeObject`, we can address it then